packages/zero-client/src/client/zero.ts

```diff
@@ -465,10 +470,13 @@ export class Zero<
 
     this.storageKey = storageKey ?? '';
 
+    const {clientSchema, hash} = clientSchemaFrom(schema);
+    this.#clientSchema = clientSchema;
+
     const replicacheOptions: ReplicacheOptions<WithCRUD<MutatorDefs>> = {
-      // The schema stored in IDB is dependent upon both the application schema
+      // The schema stored in IDB is dependent upon both the ClientSchema
```

Comment:
We could reuse the IDB cache if the schema change just added new tables... but its probably not worth the complexity.